### DISCUSSION TITLE
Increase the certificate validity to more than 24 hours

### DISCUSSION THREAD

**Discussion by Daniel-Nashed** [smallstep/certificates#401] [Q&A] [answered]
I didn't find a way to change the value to more than the [default value](https://github.com/smallstep/certificates/blob/master/docs/defaults.md#leaf-certificate---end-entity-certificate-certificates-returned-by-the-ca)  24 hours for ACME requests.

Can someone point me to the right direction?

For an internal test environment where we have to distribute the certificates I would like to use a least 30 days for the web servers.

Thanks for any tip! 
I have been searching the project without luck so far.

-- Daniel

**Comment by Daniel-Nashed** [comment, marked as answer]
OK found it [here](https://github.com/smallstep/certificates/blob/master/docs/provisioners.md#acme)

The link in this section to "step-ca ACME tutorial " is not working. Is the documentation available somewhere?

**Reply by maraino** [reply]
Hi @Daniel-Nashed we're in the process of releasing new docs, I think that doc lives right now on https://app.smallstep.com/docs/tutorials/acme-challenge, but I'm not sure if that would be the permanent location.

By the way, right now that you also need to increase `maxTLSCertDuration` if you increase `defaultTLSCertDuration`

For 30 days you need to add the claims:
```json
{
    "defaultTLSCertDuration": "720h",
    "maxTLSCertDuration": "720h"
}
```
